projects/Mallard/ios/Podfile.lock

```diff
@@ -192,7 +192,7 @@ PODS:
     - React
   - react-native-in-app-utils (6.0.2):
     - React
-  - react-native-netinfo (5.6.2):
+  - react-native-netinfo (5.3.3):
```

Comment:
Needs a rebase

Reply:
This was the version xcode has decided is the most appropriate give its dependencies etc. Do we need 5.6.x?

Reply:
Ignore me

Reply:
I can see in the commit logs it was bumped specifically. I'll make sure that change persists.